test-requirements.txt

```diff
@@ -0,0 +1 @@
+odoo14-addon-product_packaging_type @ git+https://github.com/trobz/product-attribute.git@14.0-mig-product_packaging_type#subdirectory=setup/product_packaging_type
```

Comment:
Depdendency merged: https://github.com/OCA/product-attribute/pull/760, to remove

Reply:
@phuctranerp ping

Reply:
@simahawk , @sebalix, Thanks for your review. I will work on this.